assets/rich_text_components_definitions.ts

```diff
@@ -177,7 +177,7 @@ export = {
       "description": "The custom svg diagram filename",
       "schema": {
         "type": "custom",
-        "obj_type": "SvgFilename"
+        "obj_type": "LiterallyCanvasDiagram"
```

Comment:
Why this change? The backend schema cares only about what is stored in the backend, so is it important that the svg is a LiterallyCanvasDiagram or not?

Asking because I want to understand what the plan is if we ever decide to switch from LC to another editor. Will all existing objects of this form then be incompatible with the new editor?

Reply:
The object type is used by the rte to fetch the correct component.ts file for the modal. So that's the reason why I used LiterallyCanvasDiagram because it connects literally-canvas-diagram-editor.component.ts file.

Reply:
My point still stands, though. I don't think you've addressed it, especially the second paragraph (which also explains why it's better to base this on the object type, not the editor you're using to edit the object).

Reply:
I have changed it to SvgFilename. The object is only a svg file name so it is completely compatible with any new editor.

Reply:
Done.